terraform/environments/analytical-platform-ingestion/transfer-family/security-groups.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_security_group" "transfer_server" {
+  description = "Security Group for Transfer Server"
+  name        = "transfer-family-server"
+  vpc_id      = data.aws_vpc.isolated.id
+  tags        = local.tags
+}
```

Comment:
We _could_ replace this and the rule below with https://registry.terraform.io/modules/terraform-aws-modules/security-group/aws/latest